docs/DeveloperGuide.md

```diff
@@ -234,6 +234,50 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
+### Tag adding feature
+
+### Implementation
+
+The tag adding mechanism is facilitated bt `TagAddCommand` and `TagAddCommandParser`.
```

Comment:
`bt`?

Reply:
Hmm. Looks like my commit from my laptop didn't get through. I'll make the changes.